demos/common/python/models/image_model.py

```diff
@@ -0,0 +1,69 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from .model import Model
+from .utils import resize_image, resize_image_with_aspect, resize_image_letterbox, pad_image
+
+
+class ImageModel(Model):
+    RESIZE_TYPES = {
```

Comment:
resize can be reused in video model in future, possibly better to store mapping outside model class

Reply:
Somehow done.